diff --git a/node_modules/react-native-image-picker/ios/ImagePickerManager.h b/node_modules/react-native-image-picker/ios/ImagePickerManager.h
index aa4c46a..80ff07d 100644
--- a/node_modules/react-native-image-picker/ios/ImagePickerManager.h
+++ b/node_modules/react-native-image-picker/ios/ImagePickerManager.h
@@ -29,7 +29,7 @@ RCT_ENUM_CONVERTER(
 #ifdef RCT_NEW_ARCH_ENABLED
 
 #import "RNImagePickerSpec.h"
-@interface ImagePickerManager : NSObject <RNImagePickerSpec>
+@interface ImagePickerManager : NSObject <NativeImagePickerSpec>
 @end
 
 #else
